### DISCUSSION TITLE
Mint-UI

### DISCUSSION THREAD

**Discussion by FlueschPluesch** [metaplex-foundation/metaplex-program-library#1169] [Q&A]
Hello, what is the recommended mint-ui for cmv3 at the moment, as the old UI no longer works?

**Comment by MarkSackerberg** [comment]
There is no official recommended mint UI, but there are some out there made by the community.
I personally would use https://github.com/MarkSackerberg/umi-cmv3-ui-inofficial (but I might be biased)
